doc/tracepath.xml

```diff
@@ -75,6 +75,17 @@ xml:id="man.tracepath">
     history.</para>
   </refsection>
 
+  <refsection xml:id="destination">
```

Comment:
I think I would drop this patch. The first two options documented in the manpage are -4 and -6 so it is already quite clear that tracepath supports ipv4 and ipv6.